src/telegram.ts

```diff
@@ -38,32 +35,6 @@ export class Telegram {
     return this.callApi('getFile', { file_id: fileId })
   }
 
-  /**
-   * Get download link to a file.
-   */
-  async getFileLink(fileId: string | tg.File) {
```

Comment:
Are there any blockers to implement `downloadFile()` #1703?

Reply:
I wanna defer dealing with Client until transformers are being implemented.

Reply:
If `this.#client.token` is still available, and `downloadFile` is not implemented yet - why remove `getFileLink` already?

Also, how do you see it being implemented with transformers?

Reply:
Both are simply heavily tied to Client. I removed it cuz I want to get to 0 type errors first.